content/src/filings/annual-report.md

```diff
@@ -19,12 +19,13 @@ callToActionText: File Your Annual Report
 - Names and addresses of all business partners
 - Applicable fees paid
 
-:::callout{ header="" icon="true" calloutType="conditional" }
+:::callout{ header="" icon="false" calloutType="conditional" }
 
 - Confirmation of submission of your annual report
-  :::
 
-:::callout{ header="What if I do not complete my report?" icon="true" calloutType="conditional" }
+:::
+
+:::callout{ header="What if I do not complete my report?" icon="false" calloutType="conditional" }
 After two missed annual reports, you may lose your authority to do business in New Jersey, your `Business Registration Certificate|business-reg-certificate\` may be invalidated, and you could lose access to state services, including funding opportunities and licenses.
 
 To be reinstated, businesses will need to file and pay for past reports as well as pay a reinstatement fee.
```

Comment:
[boulder] bullets don't need a space for some reason, but prose needs a leading space.

I didn't make the rules. I just stumbled upon them by accident.

```suggestion
:::callout{ header="What if I do not complete my report?" icon="false" calloutType="conditional" }
 After two missed annual reports, you may lose your authority to do business in New Jersey, your `Business Registration Certificate|business-reg-certificate\` may be invalidated, and you could lose access to state services, including funding opportunities and licenses.

 To be reinstated, businesses will need to file and pay for past reports as well as pay a reinstatement fee.
```